smartsim/experiment.py

```diff
@@ -602,10 +603,12 @@ def create_model(
         :return: the created ``Model``
         :rtype: Model
         """
-        path = init_default(getcwd(), path, str)
-
+        if name is None:
+            raise AttributeError("Entity has no name. Please set name attribute.")
+        path = init_default(osp.join(self.exp_path, name), path, str)
         if path is None:
-            path = getcwd()
+            path = osp.join(self.exp_path, name)
+
```

Comment:
I think these lines were already redundant, since path will always be something not None